model/database/common.yaml

```diff
@@ -21,3 +21,24 @@ groups:
           The `error.type` SHOULD match the error code returned by the database or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.cosmos.minimal
+    type: attribute_group
+    brief: 'Cosmos DB attributes'
+    attributes:
+      - ref: db.system
+        requirement_level: required
+      - ref: db.collection.name
+        requirement_level: required
+      - ref: db.namespace
+        requirement_level: required
+      - ref: db.operation.name
+        requirement_level: required
+      - ref: server.address
```

Comment:
server_address never exists for direct mode on operation level - unless you ant to use the server.address attribute to convey the account name?

Reply:
yes, it will be account name, something like `<database account name>.document.azure.com`